### ISSUE TITLE
Error editing newly created user groups

### ISSUE THREAD

**Issue by nmpetkov** [zikula/core#2294]
How to reproduce:
1. Go to Groups in Admin panel of the site.
2. Create a new group.
3. Try to edit it. I receive an error:
```
Oops! An Error Occurred
The server returned a "404 Not Found".
Something is broken. Please let us know what you were doing when this error occurred. We will fix it as soon as possible. Sorry for any inconvenience caused.

Sorry! No such group found.
```
On the address bar of the browser:
http://xyz.com/groups/admin/modify/6

[Zikula 1.4.0 dev last from Github]

For pre-installed groups all is OK.

**Comment by craigh** [comment]
if you switch custom_parameters.yml `env=dev` you will get a longer trace

**Comment by nmpetkov** [comment]
Obtained he reason, please check this line:
https://github.com/zikula/core/blob/1.4/src/system/Zikula/Module/GroupsModule/Api/UserApi.php#L136

**Comment by craigh** [comment]
closed by #2295
